### ISSUE TITLE
ios18 can not play audio The voice can be played on safari webkit 

### ISSUE THREAD

**Issue by likenamehaojie** [langgenius/dify#8567]
### Self Checks

- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

error 

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

ios 18 can not play audio 

### ✔️ Expected Behavior

11111

### ❌ Actual Behavior

111111

**Comment by likenamehaojie** [comment]
<img width="686" alt="image" src="https://github.com/user-attachments/assets/57ca466e-8d5e-4228-8611-c6f86fc515bb">
ios 17 also has this bug

**Comment by likenamehaojie** [comment]
the version 0.8.3 also has same bug

**Comment by crazywoola** [comment]
Link https://github.com/langgenius/dify/issues/5617
Link https://github.com/langgenius/dify/issues/7492

**Comment by likenamehaojie** [comment]
@Crazywoola, I looked at this, but my problem is that there is an issue with iOS 17.1+Safari browser on my iPhone,
The prompt is that DataMediaSource does not have endOfStream() function,
When I commented out this line, it still cannot be played,
I refer to the description of DataMediaSource by W3C.
I made the following settings for the audio when building a DataMediaSource
this.audio.disableRemotePlayback = true;
this.audio.controls = true;
At this time, it can be played, but the first and second messages of each conversation cannot be played. The voice can be played on the third message. This is the work I have done so far
The current expectation is that the first and second messages can be played normally
